account_invoice_reorder_lines/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Invoice lines with sequence number
+==================================
+
+This module was written to provide a new field on the invoice line form, allowing to manage the invoice lines
```

Comment:
"new field on the invoice line form" better change to "new field on the invoice form" because sequence is shown on invoice form, in invoice line tree.